source/module_base/module_fft/fft_base.h

```diff
@@ -0,0 +1,80 @@
+#include <complex>
+#include <string>
+#include "fftw3.h"
+#ifndef FFT_BASE_H
+#define FFT_BASE_H
+template <typename FPTYPE>
+class FFT_BASE
+{
+public:
+
+	FFT_BASE();
+	virtual  ~FFT_BASE(); 
+	
+	// init parameters of fft
+	virtual void initfft(int nx_in, int ny_in, int nz_in, int lixy_in, int rixy_in, int ns_in, int nplane_in, 
```

Comment:
one variable for a line

Reply:
Sure, comments have been added to both functions and variables.